src/SdkCommon/ClientRuntime/ClientRuntime/RestExceptionGeneric.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+using System;
+using System.Runtime.Serialization;
+namespace Microsoft.Rest
+{
+#if FullNetFx
+    [Serializable]
+#endif
+    public class RestException<V> : System.Exception, IRestException
+    {
+        
+        /// <summary>
+        /// Initializes a new instance of the ErrorModelException class.
+        /// </summary>
+        public RestException()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the ErrorModelException class.
+        /// </summary>
+        /// <param name="message">The exception message.</param>
+        public RestException(string message)
+            : base(message, null)
+        {
+        }
+
+        public RestException(string message, System.Exception innerException)
+        : base(message, innerException)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the CloudException class caused by another exception.
+        /// </summary>
+        /// <param name="message">A description of the error.</param>
+        /// <param name="requestMessage">The request message.</param>
+        /// <param name="responseMessage">The response message.</param>
+        public RestException(string message, HttpRequestMessageWrapper requestMessage, HttpResponseMessageWrapper responseMessage, int statusCode) : base(message)
+        {
+            Request = requestMessage;
+            Response = responseMessage;
+            ResponseStatusCode = statusCode;
+        }
+
+        public V Body { get; private set; }
```

Comment:
Change Body to ErrorModel or ResponseErrorModel. Bascially idea is to use the same name that we use during swagger review.
Remove private setter (basically nothing that you gain out of making it private) if made public also aligns with CloudException body property